features/page_objects/claim_summary_page.rb

```diff
@@ -1,7 +1,7 @@
 class ClaimSummaryPage < BasePage
   set_url "/external_users/claims/{claim_id}/summary"
 
-  element :continue, ".form-buttons > .govuk-button:nth-of-type(1)"
+  element :continue, "main .govuk-button-group > .govuk-button:nth-of-type(1)"
```

Comment:
**nit** do we need to scope it to `main`?

```suggestion
  element :continue, ".govuk-button-group > .govuk-button:nth-of-type(1)"
```

Reply:
Yes. when this page starts rendering, it shows cookie banner that has the same button group.